bin/pt-stalk

```diff
@@ -459,7 +458,6 @@ _parse_command_line() {
          else
             spec=$(grep "^short form:-$opt\$" "$PT_TMPDIR"/po/* | cut -d ':' -f 1)
             if [ -z "$spec"  ]; then
-               option_error "Unknown option: $real_opt"
```

Comment:
why is this removed?

Reply:
This is fix for https://jira.percona.com/browse/PT-1720 that was not added to pt-stalk, because modules were not updated properly